hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -984,7 +983,7 @@ private Builder(ConfigurationSource conf, ClientProtocol proxy) {
     }
 
     public Builder setMetadata(Map<String, String> metadata) {
-      this.metadata = metadata;
+      addAllMetadata(metadata);
```

Comment:
Nit: Should not affect anything, but the semantic seems to be changed changed from "replacing" to "adding". Might need to clear the metadata before adding it or can change the method name to `addAllMetadata` (but `OzoneBucket` is client-facing class so I don't suggest this).

I'm OK if you decide not to change it at all.